python/Ganga/GPIDev/Adapters/IBackend.py

```diff
@@ -72,23 +72,20 @@ def _parallel_submit(self, b, sj, sc, master_input_sandbox, fqid, logger):
         try:
             sj.updateStatus('submitting')
             if b.submit(sc, master_input_sandbox):
-                sj.updateStatus('submitted')
                 sj.info.increment()
+                return 1
             else:
                 raise IncompleteJobSubmissionError(fqid, 'submission failed')
         except Exception as err:
-            #from Ganga.Utility.logging import log_user_exception
-            sj.updateStatus('failed')
```

Comment:
Shouldn't this update the master job to failed if some part of the whole set of jobs fails to submit?

Reply:
Yes but not here. That is taken care of in Job.py which catched the exception raised here.